Formula/t/tfproviderlint.rb

```diff
@@ -4,6 +4,7 @@ class Tfproviderlint < Formula
   url "https://github.com/bflad/tfproviderlint/archive/refs/tags/v0.29.0.tar.gz"
   sha256 "60427ce6952106ba6c321555352645f051781ff55fe876b591e5dd2454852692"
   license "MPL-2.0"
+  revision 1
```

Comment:
Do users need to install the new build, or will whatever they have installed previously still work?

If it's the latter we can omit the `revision` bump.

Reply:
yeah, I just tested, the existing binary works fine with the latest go

Reply:
also mark the PR as no-bottle build